fineract-provider/src/main/java/org/apache/fineract/adhocquery/service/AdHocWritePlatformServiceJpaRepositoryImpl.java

```diff
@@ -137,7 +137,7 @@ public CommandProcessingResult disableAdHocQuery(Long adHocId) {
              * Checking the adhocquery present in DB or not using adHocId
              */
             final AdHoc adHoc = this.adHocRepository.findById(adHocId).orElseThrow(() -> new AdHocNotFoundException(adHocId));
-            adHoc.disableActive();
+            adHoc.setActive(true);
```

Comment:
Shouldn't this be "adHoc.setActive(false);"? The word "disable" in the original code would suggest that...

Reply:
I just checked the original code... let's leave it as is... you converted it correctly... the old naming of the functions was completely misleading.